src/TestableIO.System.IO.Abstractions/IDirectory.cs

```diff
@@ -45,105 +89,112 @@ public interface IDirectory
         string[] GetFiles(string path, string searchPattern, SearchOption searchOption);
+
 #if FEATURE_ENUMERATION_OPTIONS
-        /// <inheritdoc cref="Directory.GetFiles(string,string,EnumerationOptions)"/>
+        /// <inheritdoc cref="Directory.GetFiles(string, string, EnumerationOptions)" />
         string[] GetFiles(string path, string searchPattern, EnumerationOptions enumerationOptions);
 #endif
-        /// <inheritdoc cref="Directory.GetFileSystemEntries(string)"/>
+
+        /// <inheritdoc cref="Directory.GetFileSystemEntries(string)" />
         string[] GetFileSystemEntries(string path);
-        /// <inheritdoc cref="Directory.GetFileSystemEntries(string,string)"/>
+
+        /// <inheritdoc cref="Directory.GetFileSystemEntries(string, string)" />
         string[] GetFileSystemEntries(string path, string searchPattern);
-        /// <inheritdoc cref="Directory.GetFileSystemEntries(string,string,SearchOption)"/>
+
+        /// <inheritdoc cref="Directory.GetFileSystemEntries(string, string, SearchOption)" />
         string[] GetFileSystemEntries(string path, string searchPattern, SearchOption searchOption);
-        /// <inheritdoc cref="Directory.GetLastAccessTime"/>
+
+#if FEATURE_ENUMERATION_OPTIONS
+        /// <inheritdoc cref="Directory.GetFileSystemEntries(string, string, EnumerationOptions)" />
+        string[] GetFileSystemEntries(string path, string searchPattern, EnumerationOptions enumerationOptions);
+#endif
+
+        /// <inheritdoc cref="Directory.GetLastAccessTime(string)" />
         DateTime GetLastAccessTime(string path);
-        /// <inheritdoc cref="Directory.GetLastAccessTimeUtc"/>
+
+        /// <inheritdoc cref="Directory.GetLastAccessTimeUtc(string)" />
         DateTime GetLastAccessTimeUtc(string path);
-        /// <inheritdoc cref="Directory.GetLastWriteTime"/>
+
+        /// <inheritdoc cref="Directory.GetLastWriteTime(string)" />
         DateTime GetLastWriteTime(string path);
-        /// <inheritdoc cref="Directory.GetLastWriteTimeUtc"/>
+
+        /// <inheritdoc cref="Directory.GetLastWriteTimeUtc(string)" />
         DateTime GetLastWriteTimeUtc(string path);
-        /// <inheritdoc cref="Directory.GetLogicalDrives"/>
+
+        /// <inheritdoc cref="Directory.GetLogicalDrives()" />
         string[] GetLogicalDrives();
-        /// <inheritdoc cref="Directory.GetParent"/>
-        IDirectoryInfo GetParent(string path);
-        /// <inheritdoc cref="Directory.Move"/>
+
+        /// <inheritdoc cref="Directory.GetParent(string)" />
+        IDirectoryInfo? GetParent(string path);
+
+        /// <inheritdoc cref="Directory.Move(string, string)" />
         void Move(string sourceDirName, string destDirName);
 
+#if FEATURE_CREATE_SYMBOLIC_LINK
+        /// <inheritdoc cref="Directory.ResolveLinkTarget(string, bool)" />
+        IFileSystemInfo? ResolveLinkTarget(string linkPath, bool returnFinalTarget);
+#endif
```

Comment:
Can you please move this change out to separate `feat` PR so that have only refactoring here?

Reply:
Will revert it here and add a separate PR, after the interface changes are finished...
I probably got a bit carried away here ;-)